### PR TITLE
Merge ROTATE_LEFT and ROTATE_RIGHT gestures into one ROTATE gesture

### PR DESCRIPTION
**Pull request by larsblumberg**

### CODE REVIEW

**Comment by larsblumberg** [comment]
@auras Please update the test cases using the same commit. Right now they won't compile.

**Comment by larsblumberg** [comment]
Please merge all 3 commits into a single commit as the first commit doesn't compile the tests.

**Comment by auras** [comment]
Done

File: nuimo/src/androidTest/kotlin/com/senic/nuimo/NuimoBluetoothControllerTest.kt [hunk 1]
```diff
@@ -13,6 +13,8 @@ import kotlin.concurrent.schedule
 //TODO: Try spek test framework: http://jetbrains.github.io/spek/
 //TODO: Add timeouts to each test
 class NuimoBluetoothControllerTest: NuimoDiscoveryManagerTest() {
+    val DIRECTION_CLOCKWISE = 1;
+    val DIRECTION_COUNTER_CLOCKWISE = -1;
```
Comment (larsblumberg) [inline nuimo/src/androidTest/kotlin/com/senic/nuimo/NuimoBluetoothControllerTest.kt]: - Please remove `;` and move both constants as variables (no uppercase) into the test case's method. These constants are only needed in a single test and thus shouldn't be exposed to the class. Thus names should be `directionClockwise` and `directionCounterclockwise`. (Counterclockwise is a single word ;)
- Please rename argument `swipeDirection` to `rotationDirection` and merge your changes into the existing commit.

**Comment by larsblumberg** [comment]
Good job!

**Pull request merged**